src/components/Proposer/Proposer.story.svelte

```diff
@@ -1,28 +1,17 @@
 <script lang="ts">
 	import type { Hst } from '@histoire/plugin-svelte';
 	import Proposer from './Proposer.svelte';
-	import { AssemblyName, type Assembly } from '$models/assembly';
-	import { BillProposerType } from '$models/bill';
-	import { inProgressBill } from '../../mocks/data/bill';
+	import { AssemblyName } from '$models/assembly';
 	import type { ComponentProps } from 'svelte';
+	import { movingForwardPolitician } from '../../mocks/data/politician';
+	import { movingForwardParty } from '../../mocks/data/party';
```

Comment:
What do you think about mock data that still couple to the model?

Reply:
I was thinking the same. Decided to remove them and duplicated the values instead. 🍻